e2e-tests/scheduled-backup/conf/scheduled-backup1.yml

```diff
@@ -26,5 +26,21 @@ spec:
   pmm:
     enabled: false
   backup:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelector:
+          - matchExpressions:
+            - key: backupWorker
+              operator: In
+              values:
+              - 'True'
+    tolerations:
+      - key: "backupWorker"
+        operator: "Equal"
+        value: "True"
+        effect: "NoSchedule"
+    annotations:
+      testName: scheduled-backup
```

Comment:
could we also add Resources, Labels, Annotations, NodeSelector, SchedulerName, PriorityClassName constrains?

Reply:
how do you want to add those checks?
every check shall be done on it's own cycle or one cycle for all checks?